services/navigation/builtin/builtin.go

```diff
@@ -387,7 +428,7 @@ func computeBearing(a, b float64) float64 {
 	return t
 }
 
-func (svc *builtIn) startWaypointExperimental(extra map[string]interface{}) error {
+func (svc *builtIn) startWaypointExperimental(cancelCtx context.Context, extra map[string]interface{}) {
```

Comment:
"cancelCtx" here should probably just be named "ctx" as that's our standard practice. We usually only say "cancelCtx" when we're creating a secondary context from the one that was passed in.